Index: public/js/addOnFunction.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>// Function show and hide password\r\nfunction hideAndShow() {\r\n    let x = document.getElementById(\"password\");\r\n    let y = document.getElementById(\"lock-icon\")\r\n    if (x.type === \"password\") {\r\n        x.type = \"text\";\r\n        y.className = \"fas fa-eye\"\r\n    } else {\r\n        x.type = \"password\";\r\n        y.className = \"fas fa-eye-slash\"\r\n    }\r\n}
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/public/js/addOnFunction.js b/public/js/addOnFunction.js
--- a/public/js/addOnFunction.js	(revision 2c8bc6a188de8566dcfacd2bbdaf2a81bed37d06)
+++ b/public/js/addOnFunction.js	(date 1678780944449)
@@ -1,12 +1,12 @@
 // Function show and hide password
 function hideAndShow() {
-    let x = document.getElementById("password");
-    let y = document.getElementById("lock-icon")
-    if (x.type === "password") {
-        x.type = "text";
-        y.className = "fas fa-eye"
-    } else {
-        x.type = "password";
-        y.className = "fas fa-eye-slash"
-    }
-}
\ No newline at end of file
+  let x = document.getElementById("password");
+  let y = document.getElementById("lock-icon");
+  if (x.type === "password") {
+    x.type = "text";
+    y.className = "fas fa-eye";
+  } else {
+    x.type = "password";
+    y.className = "fas fa-eye-slash";
+  }
+}
